packages/ui-kit/src/helpers/useStorybookAccessToken.ts

```diff
@@ -0,0 +1,24 @@
+import React from 'react'
+
+export const useStorybookAccessToken = (axiosInstance, accessTokenValue = '', tokenUrlValue = '') => {
+  const [accessToken, setAccessToken] = React.useState<string | undefined>(accessTokenValue)
+
+  React.useEffect(() => {
+    async function fetchToken() {
+      if (tokenUrlValue === '') {
+        return console.error('STORYBOOK_TOKEN_URL is not defined')
+      }
+      await axiosInstance.get(tokenUrlValue).then((res) => {
+        setAccessToken(res.data.access_token)
+      })
```

Comment:
Can't we use fetch here instead? What are the benefits of using axios?

Reply:
It's a matter of personal preferences. It's a Storybook dependency and will not affect the bundle size of the libs. There are also some benefits of using axios:
- Axios has built-in XSRF(Cross-Site Request Forgery) protection, while Fetch does not.
- Axios has the ability to intercept HTTP requests but Fetch, by default, does not.
- Axios allows canceling requests and request timeout but fetch does not.
- IE11 support

Also, to make it work in Storybook it's required to use an addon to handle real API calls (not mocks). [storybook-fetch-addon](https://github.com/andreasmcdermott/storybook-fetch-addon) is 4 years old vs [storybook-axios](https://github.com/faebeee/storybook-axios) is 2 years old.